src/LiveComponent/composer.json

```diff
@@ -28,6 +28,7 @@
     "require": {
         "php": ">=8.1",
         "symfony/property-access": "^5.4.5|^6.0|^7.0",
+        "symfony/stimulus-bundle": "^2.9",
```

Comment:
I know this is something that was not done before, but don't know/remember the implications...

@kbond any opinion there ?

Reply:
Maybe can we _not_ require it there, and throw an exeption on call if the component is not installed ?

Reply:
Yeah, I wondered about this also. As this is an optional feature, maybe a hard requirement is overkill... but.. I'm thinking to leave as is and remove if there is a problem.